ports/esp32/machine_dac.c

```diff
@@ -32,23 +32,33 @@
 #include "modmachine.h"
 
 #if MICROPY_PY_MACHINE_DAC
+#if SOC_DAC_SUPPORTED
```

Comment:
Instead of adding this #if, I suggest changing `mpconfigport.h` to:
```c
#ifndef MICROPY_PY_MACHINE_DAC
#define MICROPY_PY_MACHINE_DAC (SOC_DAC_SUPPORTED)
#endif
```
Then, we can remove all the `#define MICROPY_PY_MACHINE_DAC (0)` from the board specific `mpconfigboard.h` files.

Reply:
Done

Reply:
I don't see that this has been fixed?